src/theme/DocItem/Layout/index.js

```diff
@@ -1,31 +1,123 @@
-import React from 'react';
-import clsx from 'clsx';
-import {useWindowSize} from '@docusaurus/theme-common';
-import {useDoc} from '@docusaurus/theme-common/internal';
-import DocItemPaginator from '@theme/DocItem/Paginator';
-import DocVersionBanner from '@theme/DocVersionBanner';
-import DocVersionBadge from '@theme/DocVersionBadge';
-import DocItemFooter from '@theme/DocItem/Footer';
-import DocItemTOCMobile from '@theme/DocItem/TOC/Mobile';
-import DocItemTOCDesktop from '@theme/DocItem/TOC/Desktop';
-import DocItemContent from '@theme/DocItem/Content';
-import DocBreadcrumbs from '@theme/DocBreadcrumbs';
-import styles from './styles.module.css';
-import BrowserOnly from '@docusaurus/BrowserOnly';
+import React, { useState } from "react";
+import clsx from "clsx";
+import { useWindowSize } from "@docusaurus/theme-common";
+import { useDoc } from "@docusaurus/theme-common/internal";
+import DocItemPaginator from "@theme/DocItem/Paginator";
+import DocVersionBanner from "@theme/DocVersionBanner";
+import DocVersionBadge from "@theme/DocVersionBadge";
+import DocItemFooter from "@theme/DocItem/Footer";
+import DocItemTOCMobile from "@theme/DocItem/TOC/Mobile";
+import DocItemTOCDesktop from "@theme/DocItem/TOC/Desktop";
+import DocItemContent from "@theme/DocItem/Content";
+import DocBreadcrumbs from "@theme/DocBreadcrumbs";
+import styles from "./styles.module.css";
+import BrowserOnly from "@docusaurus/BrowserOnly";
 import Icon from "@material-ui/core/Icon";
 import ContributionIcon from "../../../../static/img/contribution.svg";
 
+const MyModal = (props) => {
+  const [other, setOther] = useState(false);
+  const [feedbackSubmited, setfeedbackSubmited] = useState(false);
+  const [disableButton, setDisableButton] = useState(false);
+  let title,whatWeDo,easyRadio, solvedRadio,otherRadio;
+  
+  if (!props.show) {
+    return null;
+  }
+  
+  otherRadio = "Other";
+
+  if(props.positiveFeedback){
+    title = "What do you like about this doc?";
+    easyRadio = "Easy to understand";
+    solvedRadio = "Solved my problem";
+    whatWeDo = "Let us know what we do well:";
+  }
+  else{
+    title = "What did you not like about this doc?";
```

Comment:
```suggestion
    title = "What don't you like about this doc?";
```

The first question uses the present tense, so just being consistent.

Reply:
We kinda wrote this on purpose to make the NOT more preeminent. But we can talk about this in our meetings.